src/api.yaml

```diff
@@ -152,6 +152,24 @@ enum:
         Takes the maximum of the source and destination.<br>
         `max(RGBA_src, RGBA_dst)`
 
+  buffer binding type:
+    description: >
+      Describes the address space and access mode for the buffer binding. The
+      value used must match up to the
+      [address space](%% wgsl(#address-space) %%) and
+      [access mode](%% wgsl(#memory-access-mode) %%) set
+      in the corresponding WGSL shader.
+    ref:
+      name: GPUBufferBindingType
+      anchor: "#enumdef-gpubufferbindingtype"
+    values:
+      undefined: Undefined binding.
```

Comment:
Maybe explain that this is used as a tag that the binding is not a buffer binding?